solutions/standard/main.tf

```diff
@@ -0,0 +1,242 @@
+##############################################################################
+# IBM Cloud Provider
+##############################################################################
+
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+  region           = var.region
+  ibmcloud_timeout = 60
+}
+
+##############################################################################
+
+########################################################################################################################
+# Landing Zone Locals
+########################################################################################################################
+
+locals {
+  out = replace(file("./override.json"), "mz2o-2x16", var.machine_type)
+  image = replace(local.out,"ibm-zos-2-5-s390x-dev-test-wazi-7", var.image_name)
```

Comment:
why are doing a replace here instead of just having these as default values in the override json file itself?